environment.mk

```diff
@@ -64,6 +64,8 @@ clean: ## Resets development environment.
 	@find . -depth -type d -name '*.egg-info' -not -path "*build/$(PROJECT_NAME)/*" -delete
 	@rm -rf dist/
 	@rm -f $(INSTALL_STAMP)
+	@rm -f $(COLAB_IMAGE_BUILD_STAMP)
+	@rm -rf $(ROOT_DIR)/build/
```

Comment:
Should we nuke `build` here? We already nuke on a `deepclean` if we remove here all conda env and ipython will be removed on MacOS forcing new env to be created.

Reply:
Sure